permissions/plugin-gitlab-api.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: "gitlab-api"
+github: "jenkinsci/gitlab-api-plugin"
+paths:
+- "org/jenkins-ci/plugins/gitlab-api"
```

Comment:
wrong path, this should map to `groupId` in the pom.xml

Reply:
I have updated both changes. Can you tell me why we need to set the version as SNAPSHOT by default and what is the process to be followed when changing to release version?

Reply:
I guess `maven release plugin` handles that for us in a way that when I run `mvn release:prepare release:perform -Dusername=...  -Dpassword =...` it will change SNAPSHOT version to a releasable version, tag it, deploy then again upgrades POM to next SNAPSHOT version. Please correct me if I am wrong.